pkgs/sketch_pad/lib/execution/frame_utils.dart

```diff
@@ -0,0 +1,15 @@
+import 'dart:js_interop';
+import 'dart:js_interop_unsafe';
+
+import 'package:web/web.dart';
+
+extension HTMLIFrameElementExtension on HTMLIFrameElement {
```

Comment:
Can you add docs (either on the extension or on the method) about what this is doing and the issue we're working around? Thanks!

Reply:
Really great point. Done :)